ajax/libs/g2plot/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "g2plot",
+  "filename": "dist/g2plot.js",
+  "version": "1.0.2",
```

Comment:
```suggestion
```

Please remove the `version` attr, the auto-update bot will populate this itself when it pulls in versioned library files.